src/Runner.Worker/ExecutionContext.cs

```diff
@@ -266,19 +265,6 @@ public void RegisterPostJobStep(IStep step)
             step.ExecutionContext = Root.CreateChild(_record.Id, _record.Name, _record.Id.ToString("N"), scopeName, step.Action.ContextName, logger: _logger, insideComposite: true, cancellationTokenSource: CancellationTokenSource.CreateLinkedTokenSource(_cancellationTokenSource.Token));
             step.ExecutionContext.ExpressionValues["inputs"] = inputsData;
             step.ExecutionContext.ExpressionValues["steps"] = Global.StepsContext.GetScope(step.ExecutionContext.GetFullyQualifiedContextName());
-
```

Comment:
Simplified this a little bit. See comment in CompositeActionHandler.cs

Reply:
Setting up the child execution context is now handled by `CreateEmbeddedChild` further below in this class

The logic to setup the expression values is performed by CompositeActionHandler now. That is a better place for it since it has to deal with setting up other expression values. And also is analogous to StepsRunner